epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -203,5 +225,33 @@ public MethodInfo(String name, Set<Modifier> modifiers, List<ParameterSpec> para
       this.modifiers = modifiers;
       this.params = params;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Constructor info is very basic though so they don't necessarily need to be treated the same way. it could be a clean up task in the future though